components/atom/input/src/Input/Features/Tags/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import AtomTag, {atomTagSizes} from '@schibstedspain/sui-atom-tag'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomInput'
+const CLASS_TAGS = `${BASE_CLASS}--withTags`
+const CLASS_TAGS_CONTAINER = `${CLASS_TAGS}-container`
+const CLASS_TAGS_FOCUS = `${CLASS_TAGS}--focus`
+
+// eslint-disable-next-line react/prop-types
+const AtomTagItem = ({onClose, id, ...props}) => {
+  const _onClose = e => {
+    onClose && onClose(id)
+  }
+  return <AtomTag onClose={_onClose} {...props} />
+}
+
+const TagsHoC = WrappedInput =>
+  class Tags extends React.Component {
+    static propTypes = {
+      /* tags */
+      tags: PropTypes.any,
+
+      /* tagsCloseIcon */
+      tagsCloseIcon: PropTypes.node,
+
+      /* onRemoveTag */
+      onRemoveTag: PropTypes.func,
+
+      /* onAddTag */
+      onAddTag: PropTypes.func
+    }
+
+    state = {
+      focus: false
+    }
+
+    handleRemoveTag = index => {
+      const {onRemoveTag} = this.props
+      onRemoveTag && onRemoveTag(index)
```

Comment:
Why do you think is a better option to define and execute an empty function than this "short circuit" version of an if?  Is there any issue for using this "short circuit" pattern? If there is... won't be less performant creating and executing an empty function? Just asking.

Reply:
I've always created a `utils/noop.js` function that basically does `() => {}`. Would be the same to do what @carlosvillu says. Anyway, I don't think that this would be a critical for code optimization. But IMHO since it should be a `required` prop, I would not give it a `default behaviour`